test/unit/injector-deployment.bats

```diff
@@ -189,3 +189,20 @@ load _helpers
       yq -r '.[2].value' | tee /dev/stderr)
   [ "${actual}" = "http://not-external-test-vault.default.svc:8200" ]
 }
+
+@test "injector/deployment: authPath" {
+  cd `chart_dir`
+  local object=$(helm template \
+      --show-only templates/injector-deployment.yaml  \
+      --set 'injector.authPath=auth/k8s' \
+      . | tee /dev/stderr |
+      yq -r '.spec.template.spec.containers[0].env' | tee /dev/stderr)
+
+  local actual=$(echo $object |
+     yq -r '.[3].name' | tee /dev/stderr)
+  [ "${actual}" = "AGENT_INJECT_VAULT_AUTH_PATH" ]
+
+  local actual=$(echo $object |
+      yq -r '.[3].value' | tee /dev/stderr)
+  [ "${actual}" = "auth/k8s" ]
+}
```

Comment:
Would you mind adding another test to show the default authPath is set as expected?

Reply:
yup ;)